### PR TITLE
Minor changes in PHP code

### PR DESCRIPTION
**Pull request by RyDroid**

### CODE REVIEW

File: src/main/php/PHPMD/TextUI/CommandLineOptions.php [hunk 1]
```diff
@@ -142,9 +142,11 @@ public function __construct(array $args, array $availableRuleSets = array())
         $arguments = array();
         while (($arg = array_shift($args)) !== null) {
             switch ($arg) {
+                case '--minimum-priority':
```
Comment (ravage84) [inline src/main/php/PHPMD/TextUI/CommandLineOptions.php]: While I think this is OK, it should be added to the documentation

File: src/main/php/PHPMD/TextUI/CommandLineOptions.php [hunk 2]
```diff
@@ -142,9 +142,11 @@ public function __construct(array $args, array $availableRuleSets = array())
...
                 case '--minimumpriority':
                     $this->minimumPriority = (int)array_shift($args);
                     break;
+                case '--report-file':
```
Comment (ravage84) [inline src/main/php/PHPMD/TextUI/CommandLineOptions.php]: While I think this is OK, it should be added to the documentation

File: src/main/php/PHPMD/TextUI/CommandLineOptions.php [hunk 3]
```diff
@@ -176,6 +178,8 @@ public function __construct(array $args, array $availableRuleSets = array())
                     return;
                 case '--strict':
                     $this->strict = true;
+                case '--not-strict':
```
Comment (ravage84) [inline src/main/php/PHPMD/TextUI/CommandLineOptions.php]: Should be added to the docs.

**Comment by RyDroid** [comment]
I did the split.

**Pull request closed without merging**